static/app/views/performance/traceDetails/newTraceDetailsTransactionBar.tsx

```diff
@@ -152,21 +157,46 @@ function NewTraceDetailsTransactionBar(props: Props) {
       return;
     }
     const boundingRect = element.getBoundingClientRect();
-    const offset = boundingRect.top + window.scrollY;
+    const offset = boundingRect.top + window.scrollY - MINIMAP_SPAN_BAR_HEIGHT * 5;
```

Comment:
Might want to leave a comment on what the `MINIMAP_SPAN_BAR_HEIGHT * 5` is accounting for here. I'm not super familiar with what this offset is applying to but its significance should be stated. e.g. if it's to scroll and avoid certain components vs it's arbitrary and just nicer to scroll by this amount